server/routerlicious/packages/services-shared/src/runner.ts

```diff
@@ -101,6 +103,15 @@ export function runService<T extends IResources>(
 					.use("memory")
 			: configOrPath;
 
+	const configDumpEnabled = (config.get("config:configDumpEnabled") as boolean) ?? false;
```

Comment:
Do we want this to be invoked for all services ? A lot of services do not use runService.

Reply:
What services are you referring to? I think all of them use runService, and that's why we have alerts for services restarting that include all our different services like Scribe, Deli, Alfred.

Reply:
I went through the code to check if I had missed anything but the services I checked use RunService:

Example in Alfred:
https://github.com/microsoft/FluidFramework/blob/8a12c93bb5e28849dc3110d4bda0bd21ce8a6477/server/routerlicious/packages/routerlicious/src/alfred/www.ts#L19

Example for Kafka services like Deli and Scribe:
https://github.com/microsoft/FluidFramework/blob/8a12c93bb5e28849dc3110d4bda0bd21ce8a6477/server/routerlicious/packages/routerlicious/src/kafka-service/command.ts#L29

GitRest also uses it:
https://github.com/microsoft/FluidFramework/blob/8a12c93bb5e28849dc3110d4bda0bd21ce8a6477/server/gitrest/packages/gitrest/src/www.ts#L16

As well as Historian:
https://github.com/microsoft/FluidFramework/blob/8a12c93bb5e28849dc3110d4bda0bd21ce8a6477/server/historian/packages/historian/src/www.ts#L16

but if there are others that might have not been using runService, then indeed we need to think how to enable this logic over there as well.